src/ending-punctuation-conf.js

```diff
@@ -44,7 +44,10 @@ const finnishTerms = [
 	/^kaunokki$/,
 	/^bella$/,
 	/^musa$/,
-	new RegExp('^slm/')
+	/^local$/,
```

Comment:
Remove the languages since the validator rules won't work as intended if more languages are added later on. So these lines become `/^slm\/(fin|eng|swe)$/` -> `/^slm\//`